page/index.ejs

```diff
@@ -2,7 +2,24 @@
     <head>
         <link rel="stylesheet" type="text/css" href="app.css">
         <link href="https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600,700,900" rel="stylesheet">
+        <meta charset="utf-8">
```

Comment:
```<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->```
this is a comment from bootstrap docs. Please read the docs again and change order of tags accordingly.